iOS/Multiplatform Redux Sample/Multiplatform Redux Sample/Utils/HorizontalDoubleView.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  HorizontalDoubleView.swift
+//  Multiplatform Redux Sample
+//
+//  Created by Samuel Bichsel on 09.03.21.
+//
+
+import UIKit.UIView
+
+class HorizontalDoublekView<T>: UIView where T: UIView {
```

Comment:
Couldn't you just use a `UIStackView` with `stack.distribution = .fillEqually` and keep references to to the two subviews directly in the `CollectionPointInfoView` to avoid having to cast them every time.

Reply:
Nice catch.